### DISCUSSION TITLE
Citeproc isn't able to read special unicode characters

### DISCUSSION THREAD

**Discussion by Minh-AnhHuynh** [quarto-dev/quarto-cli#3167] [Q&A] [answered]
Hello,

Using the default citeproc engine in Quarto, I have messages such as this returning as error :
`
Error reading bibliography file bibliography/vitamin-d.bib:
(line 1886, column 18):
unexpected '\8208'
`
So as expected in the line 1886 I find the unicode character 8208 which is readable by R (when opining the .bib file) but not by citeproc somehow. Also using natbib or biblatex solves the issues, however I don't like the style format used by those engines.

Is there a way to fix this issue ?

**Comment by mcanouil** [comment]
You need to provide at least a small reproducible example, without, it's unlikely it will be fixed or that the issue will be understood.
Did you try directly using Pandoc? Did you try Quarto latest pre-release version?

Bibliography style are not define by the engine but by CSL, see <https://quarto.org/docs/authoring/footnotes-and-citations.html#citation-style>.

**Comment by Minh-AnhHuynh** [comment, marked as answer]
Somehow the issue got resolved, however I got no idea how or why. I used all of the latest version of R, RStudio and Quarto. I am very confused. I can hypothesize that there was something wrong with my cmd.exe which didn't want to launch, that I resolved by removing autorun from the registry using a guide.

Also, as far as I know, the CSL file only define the bibliography style with citeproc but not with biblatex or natbib, which have to be defined using their own package settings.
